diff --git a/node_modules/react-native-flash-message/src/FlashMessage.js b/node_modules/react-native-flash-message/src/FlashMessage.js
index 29891b6..f43b46d 100644
--- a/node_modules/react-native-flash-message/src/FlashMessage.js
+++ b/node_modules/react-native-flash-message/src/FlashMessage.js
@@ -1,5 +1,5 @@
 import React, { Component } from "react";
-import { StyleSheet, TouchableWithoutFeedback, Platform, StatusBar, Animated, Image, Text, View } from "react-native";
+import { StyleSheet, TouchableWithoutFeedback, Platform, StatusBar, Animated, Image, Text, View, TouchableOpacity } from "react-native";
 import { isIphoneX, getStatusBarHeight } from "react-native-iphone-screen-helper";
 import PropTypes from "prop-types";
 
@@ -453,6 +453,7 @@ export default class FlashMessage extends Component {
    * Non-public method
    */
   pressMessage(event) {
+    console.log(event,"event node")
     if (!this.state.isHidding) {
       const { message } = this.state;
       const hideOnPress = this.prop(message, "hideOnPress");
@@ -626,6 +627,7 @@ export default class FlashMessage extends Component {
       <Animated.View pointerEvents="box-none" style={[positionStyle(styles.root, position), animStyle]}>
         {!!message && (
           <TouchableWithoutFeedback onPress={this.pressMessage} onLongPress={this.longPressMessage} accessible={false}>
+            <View>
             <MessageComponent
               position={position}
               floating={floating}
@@ -647,6 +649,7 @@ export default class FlashMessage extends Component {
               accessibilityLabel={accessibilityLabel}
               testID={testID}
             />
+             </View>
           </TouchableWithoutFeedback>
         )}
       </Animated.View>
